src/FigmaSharp.Maui.Graphics.Sample/FigmaSharp.Maui.Graphics.Sample.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFrameworks>net8.0-android;net8.0-ios;net8.0-maccatalyst</TargetFrameworks>
+		<TargetFrameworks>net9.0-maccatalyst</TargetFrameworks>
```

Comment:
The Android and iOS target could have sense maybe to deploy the tool on a table (example, an iPad) and allow to work there.

Reply:
Restored the android + ios, used net9 for them as well.